magellan-legacy/src/main/java/com/wealthfront/magellan/Screen.java

```diff
@@ -199,26 +188,41 @@ protected void onRestore(Bundle savedInstanceState) {}
   protected void onUpdateMenu(Menu menu) {}
 
   /**
-   * Called when the Activity is resumed and when the Screen is shown.
+   * Called when the Screen in shown (including on rotation).
    */
-  protected void onResume(Context context) {}
+  @Override
+  @CallSuper
+  protected void onShow(@NotNull Context context) {
```

Comment:
onShow's behaviour will be different and that is intentional, it helps model screens and steps/journeys the same way so that its easier to reason about in the future. I'm not too worried since 2.0 is gonna be a breaking change anyways.